packages/adapter/addon/index.js

```diff
@@ -321,7 +316,6 @@ export default EmberObject.extend({
     @param {Snapshot} snapshot
     @return {Promise} promise
   */
-  createRecord: null,
```

Comment:
We have to remove these because this will take precedence over prototypal methods defined on our own adapters that extend this class or user defined ones.

Reply:
Let me know if we should replace these with a function + assert or if we would be better served in the ts migration.

Reply:
Not quite sure what you mean by 
> We have to remove these because this will take precedence over prototypal methods defined on our own adapters that extend this class or user defined ones.
Can you please explain more?

Reply:
TS playground [example](https://www.typescriptlang.org/play?#code/MYGwhgzhAEAKYCcCmA7ALtA3gKGtYyYaSASksAPYIAm0AvNAOQAWAlowNzYC+22okGAGE2IWkgAexFNRjxk6LLnyFiZSjQAUASiV48yNAFcEKJgFsAntAgB3JEgzBRtCgDNo51iiSdlvXn4KFAgnemgfW2gRVjEdLkoQihAkADoQCgBzTWBUgiQiUnIqam0OIA).